### PR TITLE
DAS-2180: Build process needs build package.

### PR DESCRIPTION
**Pull request by flamingbear**
## Description

This should fix the build pacakge.

## Jira Issue ID

[DAS-2180](https://bugs.earthdata.nasa.gov/browse/DAS-2180)

## Local Test Steps

None. Just hope.

commented out some code and ran act 

```bash

[Publish Harmony Browse Image Generator (HyBIG)/build_and_publish]   ✅  Success - Main Install build package
[Publish Harmony Browse Image Generator (HyBIG)/build_and_publish] ⭐ Run Main Build hybig-py package
[Publish Harmony Browse Image Generator (HyBIG)/build_and_publish]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/5] user= workdir=
| * Creating isolated environment: venv+pip...
| * Installing packages in isolated environment:
|   - hatch-requirements-txt
|   - hatchling
| * Getting build dependencies for sdist...
| * Building sdist...
| * Building wheel from sdist
| * Creating isolated environment: venv+pip...
| * Installing packages in isolated environment:
|   - hatch-requirements-txt
|   - hatchling
| * Getting build dependencies for wheel...
| * Building wheel...
| Successfully built hybig_py-2.0.0.tar.gz and hybig_py-2.0.0-py3-none-any.whl
```

## PR Acceptance Checklist
* [n/a] Jira ticket acceptance criteria met.
* [n/a] `CHANGELOG.md` updated to include high level summary of PR changes.
* [n/a] `docker/service_version.txt` updated if publishing a release.
* [n/a] Tests added/updated and passing.
* [n/a] Documentation updated (if needed).

### CODE REVIEW

**Comment by flamingbear** [comment]
I do wonder if this won't actually publish because the version is the same as `main` 🤔

**Comment by owenlittlejohns** [comment]
Good question. I think you can manually trigger the publication workflow (the `workflow_dispatch` was added for the 1.0.0 release, I think).

You might need to delete the GitHub release, the git tag and the ghcr.io package. Now I've said all that, maybe just going to 2.0.1 is better?

**Comment by flamingbear** [comment]
Lucky for me, it didn't create a v2 release on github, there is no release or tag, just the docker image, which will get written over?  IDC I can bump the version and we can just try again. probably the easiest.

File: .github/workflows/publish_release.yml [hunk 1]
```diff
@@ -83,6 +83,11 @@ jobs:
         with:
           python-version: '3.11'
 
+      - name: Install build package
```
Comment (owenlittlejohns) [inline .github/workflows/publish_release.yml:86]: (Probably a dopey question): Does this need [the GDAL wizardry](https://github.com/nasa/harmony-browse-image-generator/blob/main/.github/workflows/run_lib_tests.yml#L26-L30) that `run_lib_tests.yml` includes?

Comment (flamingbear) [reply .github/workflows/publish_release.yml:86]: We're going to find out.  I am not an expert at package building, but my few google searches said no and when I ran it with ACT it showed that it did build the package.  Of course I didn't test that, but the package should only have information about what it needs to build not the actual libraries no?

Comment (owenlittlejohns) [reply .github/workflows/publish_release.yml:86]: Okay. Let me approve this and then we can give it a whirl!

**Review by owenlittlejohns — APPROVED** [review]
🤞

**Pull request merged**